ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/context/AppContextImpl.java

```diff
@@ -414,6 +414,10 @@ public Project getRootProject() {
             }
         }
 
+        if (currentResources[0] == null) {
```

Comment:
It means that current resource (selected resource) is null. In this class 
`currentResources[0] == currentResource`
I've made some changes and now it's more clear, I hope.